library/RDSYS

```diff
@@ -1,5 +1,9 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "XCL" BASE 10 FORMAT XCCS)
-(FILECREATED "30-Jul-2021 13:12:07" |{DSK}<Users>kaplan>Local>medley3.5>git-medley>tmp>RDSYS.;8|
+<<<<<<< Updated upstream
+(FILECREATED "19-Aug-2021 12:07:05" |{DSK}<Users>kaplan>Local>medley3.5>git-medley>tmp>RDSYS.;68|
+=======
```

Comment:
This is a consequence of all the screwing around  that it takes to work in a git hub/branch world, where it will or won't let you cache or uncache things if other things have been changed.  Sometimes you have to make interfering changes get out of the way to make progress.  Especially when fooling around with the makeinit process itself.

There is another issue about ignoring everything in the loadups directory.  That would make things like this go way.